src/main/java/duke/exceptions/MissingDeadlineParamException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exceptions;
+
+/**
+ * Custom Exception to be used when there is a Missing Parameter for the Deadline Command is encountered
+ */
+public class MissingDeadlineParamException extends DukeException {
```

Comment:
It's great that you created many different subclasses of DukeException, which definitely helps to identify what exactly is causing the problem.